pkg/k8s/namespaces/crud.go

```diff
@@ -21,3 +27,100 @@ func IsOktetoNamespace(ns string, c *kubernetes.Clientset) bool {
 	}
 	return n.Labels[OktetoLabel] == "true"
 }
+
+//CheckAvailableResources checks if therre are available resources for activating the dev env
+func CheckAvailableResources(dev *model.Dev, create bool, d *appsv1.Deployment, c *kubernetes.Clientset) error {
+	needCPU := resource.MustParse("0")
+	needMemory := resource.MustParse("0")
+	if !existsSyncthing(dev, c) {
+		needCPU.Add(resource.MustParse("500m"))
```

Comment:
can we get this values from the deployment object so we don't hard code them here?